### ISSUE TITLE
Can't update using brew

### ISSUE THREAD

**Issue by adamw** [VirtusLab/scala-cli#1879]
Tried today to get 0.2.0:

```
[~]% brew install Virtuslab/scala-cli/scala-cli
scala-cli 0.1.16 is already installed but outdated (so it will be upgraded).
==> Fetching virtuslab/scala-cli/scala-cli
==> Downloading https://github.com/Virtuslab/scala-cli/releases/download/v0.2.0/scala-cli-x86_64-apple-darwin.gz
curl: (22) The requested URL returned error: 404
```

**Comment by lwronski** [comment]
@adamw Hi, thanks for reporting. We encountered a bug during the release of version `0.2.0`, but we fixed it now and it should no longer occur. However, if you tried to upgrade scala-cli with brew before our fix to `0.2.0`, you have to run the following command to update:
```
brew upgrade Virtuslab/scala-cli/scala-cli --fetch-HEAD
```

if above command doesn't work you can try to run the following commands:
```
brew uninstall scala-cli
brew untap virtuslab/scala-cli
brew install Virtuslab/scala-cli/scala-cli
```

**Comment by adamw** [comment]
The first method did not work:

```
[~]% brew upgrade Virtuslab/scala-cli/scala-cli --fetch-HEAD
==> Upgrading 1 outdated package:
virtuslab/scala-cli/scala-cli 0.1.16 -> 0.2.0
==> Fetching virtuslab/scala-cli/scala-cli
==> Downloading https://github.com/Virtuslab/scala-cli/releases/download/v0.2.0/scala-cli-x86_64-apple-darwin.gz
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/366035268/799b1004-fd9a-447f-aad1-c69b10fde489?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJ
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 14818175903e79915fcf139fff17c12f4263dea43f0de72a6b8254bb2e8ef5c6
  Actual: 3c707fe1b4a83baf3baac814475a89a51ed7b4b5c09c330953ac68b7e09b8fdc
    File: /Users/adamw/Library/Caches/Homebrew/downloads/0adb5cd4afdd221eb72469fb0a95555c16c0ab8cdcb65419eca9eb3745a177e3--scala-cli-x86_64-apple-darwin.gz
To retry an incomplete download, remove the file above.
```

but the second did, thanks! :)
